File: Assignment_1/Assignment_1_q2.c
//2) factorial n! say for n=10 or 15, check and trap negative integers, say for -5!

#include <stdio.h>
int main() {
	//defining the variables
    int n;
    int factorial = 1;
    int i;
	//calling for the input user
    printf("The factorial of the integer \'n\' can be found out\nWhat is the n value? : ");
    scanf("%d", &n); 
    //condition for positive and negative indeger
    if(n>0){
    	//loop 
    	for(i=1; i<n+1; i++){
    		factorial *= i;
		}
	}	
	//printing the sum
	printf("The factorial of the %d is %d\n", n, factorial);
    return 0;    
}

File: Assignment_1/assignment_1_q2.py
# -*- coding: utf-8 -*-
"""
Created on Fri Aug  7 15:10:25 2020

@author: fida
"""
"""
2) factorial n! say for n=10 or 15, check and trap negative integers, say for -5!

"""
print("The factorial of integer \'n\' can be found")
factorial = 1
n = int(input("What is the \'n\' value ? "))
if n>0:
    for i in range(1,n+1):
        factorial *=i
    print (factorial)
else:
    print (factorial)